app/build.gradle

```diff
@@ -95,6 +95,8 @@ dependencies {
 
     //Timber
     implementation loggingDependencies.values()
+    implementation 'com.google.android.material:material:1.0.0'
```

Comment:
These two dependencies are already  declared in dependencies.gradle file and they are called from here. Maybe they were added by accident?